flake.nix

```
@@ -12,46 +15,33 @@
     nix-helm.inputs.nixpkgs.follows = "nixpkgs";
 
     std = {
-      url = "github:divnix/std";
+      url = "github:divnix/std/v0.24.0-1";
```

lets avoid changing STD revision, it may cause us problems

can you do this instead?
```suggestion
      url = "github:divnix/std/16edd0fc1e98c28180d845ef1011610f7b800976";
```